### DISCUSSION TITLE
While creating a questionset, kafka does not get any updates

### DISCUSSION THREAD

**Discussion by mahesh-maney** [Sunbird-coKreat/Community#57] [General Discussions]
while creating a question-set, the data does not get posted to kafka topics. Hence we are not able to list the newly created question-sets.
What we have debugged is that, while the question-set is created, logstash will post the question-set metadata to kafka topic.
Therefore none of the data is synced to elasticsearch and hence the issue.
We have redeployed the kafka servcie and logstash service couple of times. The required kafka topic exists and We were able to list all the kafka topics. However data isn't syncing up. 

CSP : GCP
Version : release-5.1.0-vdn

@Krishnaj20 @vinukumar-vs

**Comment by vinukumar-vs** [comment]
@santhosh-tg @pallakartheekreddy Any idea what can be the issue?

@mahesh-maney we are looking for more details with logs to understand the problem better.

**Comment by mahesh-maney** [comment]
@vinukumar-vs, debugging the issue, we can see the logs pertaining to questionset (wjile creating question-set). Neo4J logs aren't syncing with logstash. There are no entires in the log file of logstash.

**Comment by krgauraw** [comment]
@mahesh-maney Please check if the neo4j plugin (kernel extension) is deployed properly under neo4j plugin folder which populates data to log file for each neo4j transaction. If the jar is available,  you may try restarting neo4j once. Also Can you please share the name of log file you are referring to for this issue ?
cc: @vinukumar-vs @pallakartheekreddy

**Comment by mahesh-maney** [comment]
@krgauraw ,
initially the neo4j plugin was not deployed. We had this deployed just before you messaged us. However the same issue persisted. We restarted the search indexer flink job and then issue resolved :-)
Thank you @krgauraw appreciate your help.
